caffeineWiz.json

```diff
@@ -19,15 +19,19 @@
         "arm64",
         "arm"
     ],
-    "branch": "v0.1.0",
-    "license": "NeonAI Licensed",
-    "icon": "https://0000.us/klatchat/app/files/neon_images/icons/neon_paw.png",
-    "category": null,
+    "branch": "v0.1.1",
+    "license": "NeonAI License v1.0",
+    "icon": "logo.svg",
```

Comment:
can the logo be replaced with
`https://raw.githubusercontent.com/NeonGeckoCom/CaffeineWiz.neon/master/logo.svg`

Reply:
I'm strictly parsing that path out of the readme. My only concern with using the github path is what branch it points to, since icons theoretically could change and branches get out of sync.. I suppose this could build that path though from branch and repo parsed in the json

Reply:
you are right, this should match the branch and be 
```https://raw.githubusercontent.com/NeonGeckoCom/CaffeineWiz.neon/v0.1.1/logo.svg```

however it should not currently be only logo.svg, the url above will be automatically detected because logo is in a default location, but passing logo.svg will override that

but you do bring up a valid use case, i should augment OSM to account for relative paths in relation to root dir, that way you could set "caffeinewizlogo.svg" or anything else and url would be constructed from it

for now lets keep the full url until i send a PR to OSM

Reply:
Okay; I'll manually fix this one for now